packages/functions/events/createBirthdaySquad.ts

```diff
@@ -13,27 +15,36 @@ import { publishEvent } from "@/utils/eventBridge/publishEvent";
 export const handler = handleEvent(
   "createBirthdaySquad",
   async ({ team, birthdayPerson, eventId }) => {
+    const { KRISZTA_SLACK_USER_ID, MATE_SLACK_USER_ID } = Config;
+
     try {
       const appliedSquadMembers = await getSquadMembers(team, birthdayPerson);
 
       const squadMembers = [...appliedSquadMembers];
 
       if (appliedSquadMembers.length < MIN_BIRTHDAY_SQUAD_SIZE) {
-        const randomSquadMember = await getRandomSquadMembers({
+        const randomSquadMembers = await getRandomSquadMembers({
           teamId: team,
           usersToExclude: [birthdayPerson, ...appliedSquadMembers],
```

Comment:
Shouldn't we exclude Kriszta / Máté so we are always pulling other members in?

Reply:
🤯